src/routes/safe/components/Transactions/TxList/TxInfoSettings.tsx

```diff
@@ -30,13 +31,13 @@ export const TxInfoSettings = ({ settingsInfo }: TxInfoSettingsProps): ReactElem
     case 'REMOVE_OWNER': {
       const title = settingsInfo.type === 'ADD_OWNER' ? 'Add owner:' : 'Remove owner:'
       return (
-        <InfoDetails title={title}>
+        <InfoDetails title={title} header={<ThresholdWarning />}>
           <AddressInfo
             address={settingsInfo.owner.value}
             name={settingsInfo.owner?.name || undefined}
             avatarUrl={settingsInfo.owner?.logoUri || undefined}
           />
-          <InfoDetails title="Change required confirmations:">{settingsInfo.threshold}</InfoDetails>
+          <InfoDetails title="Increase/descrease confirmation policy to:">{settingsInfo.threshold}</InfoDetails>
```

Comment:
There is a typo here: 
```suggestion
          <InfoDetails title="Increase/decrease confirmation policy to:">{settingsInfo.threshold}</InfoDetails>
```